### DISCUSSION TITLE
Changing StatusBar color in MAUI

### DISCUSSION THREAD

**Discussion by russnash** [CommunityToolkit/Maui#390] [Q&A] [answered]
Google has shown quite a few hits for the ability to change the status bar color using the Xamarin Community Toolkit, but I have been unable to locate any information for doing same with the Maui Community Toolkit.  Does the Maui toolkit also contain this functionality?  If so, could someone please point me in the right direction?  I am looking to change status bar color on Androin and IOS by the way.

Thanks in advance!

Russ.

**Comment by TheCodeTraveler** [comment, marked as answer]
Hi Russ! We have an open Proposal for `StatusBarEffect` and we'd love your help implementing it!

Would you be able to help out? 

https://github.com/CommunityToolkit/Maui/issues/85

**Reply by russnash** [reply]
> Hi Russ! We have an open Proposal for `StatusBarEffect` and we'd love your help implementing it!
> 
> Would you be able to help out?
> 
> #85

Hi brminnick,

I would love to help out, but I'm not sure if my skill set is up to it.  I have never used Xamarin and only started using Maui about a month ago.

**Reply by pictos** [reply]
@russnash if you want you can help with samples, and if you like you can contribute with the source code in the future. Let me know what do you think?

**Reply by russnash** [reply]
> @russnash if you want you can help with samples, and if you like you can contribute with the source code in the future. Let me know what do you think?

I would definitely enjoy helping as above.

**Reply by pictos** [reply]
@brminnick do you have any open issue that needs a sample to indicate?

**Reply by TheCodeTraveler** [reply]
Of course! Both [MultiConverter](https://github.com/CommunityToolkit/Maui/issues/72) and [MathExtension](https://github.com/CommunityToolkit/Maui/issues/71) still need samples added to the `CommunityToolkit.Maui.Samples` app. 

Would you like to grab one of those? If so, I'll assign them to you!

**Reply by russnash** [reply]
Certainly!  I will be out of town from tomorrow until Wednesday evening, but I can start on them when I return.  By the way, are there any rules / guidelines etc written anywhere I should peruse first?

**Reply by TheCodeTraveler** [reply]
Great! Yup - you can find our contribution guidelines here: https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md

**Reply by pictos** [reply]
@russnash you also can take a look on samples on XCT and move them to this project or you can create a new one if you want. Here's our old sample for [MathExtensions](https://github.com/xamarin/XamarinCommunityToolkit/blob/main/samples/XCT.Sample/Pages/Converters/MathExpressionConverterPage.xaml)

**Reply by russnash** [reply]
Good morning guys, I was just wondering what I could do to help out further, but am unsure about how you like to manage that?  Should I just go through the open issues and request assignment of issues that I believe I can help out with?  How do you prefer to do things?

**Reply by pictos** [reply]
@russnash you can search for the issues labeled with `help wanted` and ask to help with it. At the main post, you will see a bunch of checkboxes, so you can do one of those items or all of them. Let me know if that doesn't make sense and if you need any further help (:

**Comment by CliffAgius** [comment]
Closed as answered
